.gitlab-ci.yml

```diff
@@ -88,7 +88,7 @@ review-app:
       ]
       EOF
       )
-    - git clone -b cmccarthy/pivcac --single-branch https://gitlab-ci-token:${CI_JOB_TOKEN}@gitlab.login.gov/lg-public/identity-idp-helm-chart.git
+    - git clone -b master --single-branch https://gitlab-ci-token:${CI_JOB_TOKEN}@gitlab.login.gov/lg-public/identity-idp-helm-chart.git
```

Comment:
we've migrated all our repos to use `main` as the default branch, is that a possibiltiy for this Gitlab project too?

Reply:
We can make that change.

Reply:
@zachmargolis Made the requested change. Default branch for the helm chart repo is now main.